bigtable/client/client_options_test.cc

```diff
@@ -18,15 +18,21 @@
 
 TEST(ClientOptionsTest, ClientOptionsDefaultSettings) {
   bigtable::ClientOptions client_options_object = bigtable::ClientOptions();
-  EXPECT_EQ("bigtable.googleapis.com", client_options_object.endpoint());
+  EXPECT_EQ("bigtable.googleapis.com", client_options_object.data_endpoint());
+  EXPECT_EQ("bigtableadmin.googleapis.com",
+            client_options_object.admin_endpoint());
   EXPECT_EQ(typeid(grpc::GoogleDefaultCredentials()),
             typeid(client_options_object.credentials()));
 }
 
 TEST(ClientOptionsTest, ClientOptionsCustomEndpoint) {
+  // TODO(#23) - setenv() is a Unix specific call ...
```

Comment:
[Abseil uses](https://github.com/abseil/abseil-cpp/blob/029795a9b5281379f892fbbe3f9a400d5a33f5cc/absl/synchronization/mutex_test.cc#L1050):

```c++
#ifdef WIN32
      SetEnvironmentVariable(...);
#else
      setenv(...);
#endif
```

Maybe it's worthwhile to start building a "common" library within `google-cloud-cpp` that will contain utility functions across products, such as a platform abstraction library and include such methods?

Reply:
Created #100 to track this.